### ISSUE TITLE
Test for modifier when listening to key events

### ISSUE THREAD

**Issue by waldbaerkoch** [frend/frend.co#75]
I would expect the components not to swallow key events with modifiers like `Alt+ArrowLeft` for history back.

**Comment by adamduncan** [comment]
Good point, thanks @waldbaerkoch. We'll look into a cross-platform/browser fix for this and make sure we're not catching unnecessary key events.

**Comment by adamduncan** [comment]
Added modifier guard clauses  in the `keydown` handlers for altKey and metaKey. In fix branch commit c44af13

Is this condition too simplistic? Keeps it concise without getting into `getModifierState` territory.

``` js
let isModifierKey = (e.metaKey || e.altKey) ? true : false;
if (isModifierKey) return;
```

Will do more cross-browser testing, and open to alternative approaches.

**Comment by adamduncan** [comment]
Just reread that and realised the ternary statement is redundant. Will refactor. :sleeping:

**Comment by adamduncan** [comment]
Tested the above approach in Chrome/FF/IE on Windows and Chrome/Safari on Mac. I'm happy it's resolved the issue.

@thomasdigby, @waldbaerkoch - are you aware of a better approach for this key event side-effect?

**Comment by waldbaerkoch** [comment]
I think, that is an appropriate solution. Thanks.

**Comment by thomasdigby** [comment]
Merge away 🏌

**Comment by adamduncan** [comment]
Closed with #77
